reference/lifecycle-terminology.md

```diff
@@ -0,0 +1,61 @@
+Software Lifecycle Terminology
+==============================
+
+We work on software that's anywhere in its lifecycle - from something that we've created just to see if it's possible, through to widely-used and well-maintained software that's used in production. 
+
+In order to be clear about what people should expect from a piece of software, we use specific terminology. This page documents our interpretation of these terms, and is intended to be referred to by developers at ODSC, developers that we're collaborating with, and clients. 
+
+Any particular piece of software might have elements that are at different stages of development. For example, a new feature in a mature software product might be a prototype, while the main software product is production-ready. 
+
+As a rule of thumb, it's approximately an order of magnitude of extra work, complexity or cost to move up each level; that is to say that a production implementation of a feature is 100x more complex than its prototype. 
+
+## Prototype
+
+A prototype has been created to see if something is technically feasible, or to demonstrate what something could look like. 
+
+It will:
+* Be quick to make
+* Serve a single purpose - for example to try out an interface idea, or to test an algorithm
```

Comment:
I think your 'for example' here is probably the only purpose (try out an idea).

Reply:
I specifically say an *interface* idea (eg the first version of the metadata presentation in 360 CoVE). I think that the algorithm point still stands - maybe we write a commandline tool that takes a spreadsheet and outputs some analysis, as the prototype of a future web app? 

Or, would that not be a prototype, in your view?